### DISCUSSION TITLE
How do I access an external USB drive?

### DISCUSSION THREAD

**Discussion by Clownfish33** [Rolf-Hempel/PlanetarySystemStacker#72] [General]
I have large video files on an external USB drive attached to my MacBook.  
But I cant seem to find a way to navigate to it.    The /directory for the drive does not show up in PSS, but is mounted on my desktop.  What's the trick?

**Comment by Planet30** [comment]
Go to /Volumes/ folder. It should be mounted there.

**Comment by Clownfish33** [comment]
I did.. but strangely it wasn't.  It's mounted, and I can access the files, but I cant see it in PSS.I wonder if its a permissions issue.


    On Sunday, August 13, 2023, 10:55:25 AM MDT, Planet30 ***@***.***> wrote:  
 
 


Go to /Volumes/ folder. It should be mounted there.

**Reply by Planet30** [reply]
Did you try CMD-i on the drive icon on the desktop? That should show you the path.

Sometimes it also helps to relaunch the finder with CMD-Option-ESC.

**Reply by Planet30** [reply]
You can check and change the permissions with cmd-i on the drive icon.

More likely:  Python probably does not have permission to certain folders and section on your Mac. This will not change by reformatting your external drive! This is a feature in macOS. App-dependent access to your file system. 

Check in security and privacy settings of your Mac which app has access to what.  

This is related to your Python installation and not PSS directly. 

Check here:
https://support.apple.com/en-gb/guide/mac-help/mchld5a35146/mac

**Comment by Clownfish33** [comment]
Yes.  Still not working.  I am pretty certain the issue is permissions.  There’s a discussion of what I am seeing on Apple’s website.  The only way I can fix it is by reformatting the drive.

**Comment by Clownfish33** [comment]
Thats the bug I was talking about - I cant change the perms!The permission shows "You have custom access" and will not let me change it.


    On Monday, August 14, 2023, 12:21:25 AM MDT, Planet30 ***@***.***> wrote:  
 
 


You can check and change the permissions with cmd-i on the drive icon.

More likely: Python probably does not have permission to certain folders and section on your Mac. This will not change by reformatting your external drive! This is a feature in macOS. App-dependent access to your file system.

Check in security and private setting of your Mac which app has access to what.

This is related to your Python installation and not PSS directly.

Check here:
https://support.apple.com/en-gb/guide/mac-help/mchld5a35146/mac

**Comment by Clownfish33** [comment]
Here's the Info box.  I can't change the permissions from this.




    On Monday, August 14, 2023, 12:21:25 AM MDT, Planet30 ***@***.***> wrote:  
 
 


You can check and change the permissions with cmd-i on the drive icon.

More likely: Python probably does not have permission to certain folders and section on your Mac. This will not change by reformatting your external drive! This is a feature in macOS. App-dependent access to your file system.

Check in security and private setting of your Mac which app has access to what.

This is related to your Python installation and not PSS directly.

Check here:
https://support.apple.com/en-gb/guide/mac-help/mchld5a35146/mac

**Comment by Clownfish33** [comment]
Yeah!  I saw messages on CN which suggested I drag and drop - and that worked!  Also, after I did that the regular Open Dialog now shows the drive!
Peter


 
    On Monday, August 14, 2023, 12:21:25 AM MDT, Planet30 ***@***.***> wrote:  
 
 


You can check and change the permissions with cmd-i on the drive icon.

More likely: Python probably does not have permission to certain folders and section on your Mac. This will not change by reformatting your external drive! This is a feature in macOS. App-dependent access to your file system.

Check in security and private setting of your Mac which app has access to what.

This is related to your Python installation and not PSS directly.

Check here:
https://support.apple.com/en-gb/guide/mac-help/mchld5a35146/mac

**Comment by Planet30** [comment]
That's a Mac.  The solution is too easy and too natural.  We were overthinking it 😃👍 Congratulations!
